mreg/models.py

```diff
@@ -769,7 +769,7 @@ class History(models.Model):
     model_id = models.PositiveIntegerField()
     model = models.CharField(max_length=64)
     action = models.CharField(max_length=64)
-    data = pgfields.JSONField()
+    data = models.JSONField()
```

Comment:
Changing the field requires you to [make a migration](https://docs.djangoproject.com/en/3.1/ref/django-admin/#django-admin-makemigrations)

Reply:
Good catch; I had already made it but forgot to commit it.